src/client.ts

```diff
@@ -3811,7 +3811,7 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
             algorithm.free();
         }
 
-        await this.importRoomKeys(keys, {
+        await this.getCrypto()?.importRoomKeys(keys, {
```

Comment:
note that this method still won't work with rust crypto due to the test of `this.crypto` at the top. Arguably fixing this method is out of scope for this PR though.